gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/CompactionJobConfigurator.java

```diff
@@ -249,6 +250,10 @@ protected boolean configureInputAndOutputPaths(Job job, FileSystemDataset datase
     CompactionPathParser.CompactionParserResult rst = parser.parse(dataset);
     this.mrOutputPath = concatPaths(mrOutputBase, rst.getDatasetName(), rst.getDstSubDir(), rst.getTimeString());
 
+    if(this.state.contains(ConfigurationKeys.USE_DATASET_LOCAL_WORK_DIR)) {
+      mrOutputBase = this.state.getProp(MRCompactor.COMPACTION_DEST_DIR);
+      this.mrOutputPath = concatPaths(mrOutputBase, rst.getDatasetName(), ConfigurationKeys.TMP_DIR, rst.getDstSubDir(), rst.getTimeString());
```

Comment:
Seems too long a line, did you use the style file in https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/ ?

Reply:
CI process should check the code style, looks like it's not working as expected. Worth checking why it's not raising an error.

Reply:
Yes. I do use the linkedin gobblin code style